reportportal/v5/values.yaml

```diff
@@ -132,7 +131,7 @@ elasticsearch:
   endpoint: http://elasticsearch-master:9200
 
 minio:
-  secretName: ""
+  SecretName: ""
```

Comment:
this is an issue, but changing the value will break existing installations. let's keep the value as is

Reply:
I have updated the references to minio.secretname to minio.SecretName and versioned this chart as to not break future installations.

Reply:
@avarabyeu any additional updates needed?